manage.py

```diff
@@ -48,17 +48,27 @@ def add_event_identifier():
 
 @manager.command
 def fix_event_and_speaker_images():
-    events = Event.query.filter(Event.original_image_url.isnot(None),
-                                or_(Event.thumbnail_image_url == None, Event.large_image_url == None,
-                                    Event.icon_image_url == None)).all()
+    events = Event.query.filter(
+        Event.original_image_url.isnot(None),
+        or_(
+            Event.thumbnail_image_url == None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4884175)